### ISSUE TITLE
Public broadcaster (ERR) APIs

### ISSUE THREAD

**Issue by tambeta** [okestonia/Estonian-Open-Data-Issue-Tracker#50]
A bit of a pet peeve of mine is that the all the \[meta\]data produced daily by the public broadcaster (ERR) is not programmatically accessible. Or rather, it is in many cases, but hidden behind private, undocumented APIs. For example, Raadio 2 (likely analogous for other channels) has API endpoints to fetch [air times for a particular programme](http://r2.err.ee/api/loader/airtimesforprogram/vibratsioon?channel=raadio2&ShowAll=False) or the [schedule for a particular date](http://otse.err.ee/api/schedule/GetTimelineDay?channel=r2&year=2017&month=3&day=9). I have reverse engineered these for my purposes; when I enquired Raadio 2 about opening up / documenting the APIs, I got a terse response about these being for private use.

Basically, the means to fetch machine-readable data are already there, it would simply take some effort on ERR's part to document the APIs and maintain them so that they would be stable and usable for 3rd parties. This is where OK Estonia's lobby could help get things moving.

Example use case: [a script to download shows from R2](https://github.com/tambeta/scripts/blob/master/eesti/r2get.py). Having open APIs and a welcoming culture for 3rd party developers would certainly create synergy here.

**Comment by andreskytt** [comment]
Understood. @infokujur, please add the ERR label

**Comment by andreskytt** [comment]
Discussions with ERR are ongoing. Currently, it would seem it is a legal issue from their side - the data is meant to be used for non-commercial use only and compliance is achieved via unintentional obfuscation.

**Comment by keeganmcbride** [comment]
https://www.err.ee/sitemap/videos.xml here is some of the open data for videos on ERR
https://kultuur.err.ee/sitemap/news.xml

Some XML stuff here, not surehow useful.

**Comment by infokujur** [comment]
Rahvusringhäälingu andmeid ei ole opendata.riik.ee lehel viidatud. https://opendata.riik.ee/teabevaldajad/ Palume olemasolevad avaandmed sinna viidata.

Ja ühtlasi kustutan dublikaat sildi ERR.
